hr_timesheet_holiday/models/hr_leave_type.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 Sunflower IT <http://sunflowerweb.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrLeaveType(models.Model):
+    """Add project to holiday status"""
+
+    _inherit = "hr.leave.type"
+
+    analytic_account_id = fields.Many2one(
+        comodel_name="account.analytic.account",
+        string="Analytic Account",
+        related="project_id.analytic_account_id",
+    )
+    project_id = fields.Many2one(
```

Comment:
I think we should depend this module on the standard module project_timesheet_holidays and reuse the field timesheet_project_id 

https://github.com/odoo/odoo/blob/15.0/addons/project_timesheet_holidays/models/hr_holidays.py#L22